dialog/it-it/my address on X is Y.dialog

```diff
@@ -0,0 +1 @@
+Sulla interfaccia{{interface}} il mio indirizzo I.P è {{ip}}.
```

Comment:
I think there's a space missing here (before `{{interface}}`). Can you do a quick fix? Or maybe it's supposed to be this way?